modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/config/initialState.es.js

```diff
@@ -132,9 +109,7 @@ export const initState = (
 	},
 	{view}
 ) => {
-	const pages = initialPages.length
-		? normalizePages(initialPages)
-		: INITIAL_PAGES;
+	const pages = normalizePages(initialPages);
```

Comment:
Have you changed `normalizePages` impl? Why are we removing this assert for adding the default value?